draft-ietf-jsonpath-base.md

```diff
@@ -1577,27 +1660,26 @@ expression, which is in {{-iregexp}} form.
 $[?search(@.author, "[BR]ob")]
 ~~~
 
-Its first argument is an optional string that is searched for at least one
-substring that matches the iregexp contained in the string
-that is the second argument.
-The result is `true` if such a substring exists, `false` otherwise.
-
-The result is `Nothing` if the first argument is not a string or
-the second argument is not a string conforming to {{-iregexp}}.
+Its arguments are instances of ValueType.
+If the first argument is not a string or the second argument is not a
+string conforming to {{-iregexp}}, the result is `LogicalFalse`. <!-- XXX -->
+Otherwise, the string that is the first argument is searched for at
+least one substring that matches the iregexp contained in the string
+that is the second argument; the result is `LogicalTrue` if
```

Comment:
The above sentence was truncated.

Reply:
```suggestion
that is the second argument; the result is `LogicalTrue` if
 the string contains a match the iregexp and `LogicalFalse` otherwise.
```

(referenced from `match()` definition)

Reply:
Fixed in 3e1265a